django_mock_queries/query.py

```diff
@@ -266,16 +266,23 @@ def update(self, **attrs):
         return count
 
     def _delete_recursive(self, *items_to_remove, **attrs):
+        sum_deleted = 0
+        items_deleted = defaultdict(int)
+
         for item in matches(*items_to_remove, **attrs):
             self.items.remove(item)
             self.fire(item, self.EVENT_DELETED)
+            sum_deleted += 1
+            items_deleted[item._meta.label] += 1
 
         if self.clone is not None:
             self.clone._delete_recursive(*items_to_remove, **attrs)
 
+        return sum_deleted, items_deleted
```

Comment:
you dont need to store a sum_deleted variable.
You can just use `return sum(items_delete.values()), items_deleted`